docker/dev_entrypoint.sh

```diff
@@ -6,6 +6,9 @@ set -e
 NETWORK_ID=${NETWORK_ID:-420}
```

Comment:
TODO: add these changes to the other files `docker/*` and standardize on a good poll interval. I think between 0.5-1 second is a good time, leaning towards 1 second